development/image-url-helper/README.md

```diff
@@ -60,3 +71,17 @@ See the list of flags available for the `list` command:
 | **--resources-directory** |   Yes    | Path to the Kyma resources directory.|
 | **--output-format**       |    No    | Name of the output format (JSON/YAML).|
 | **--exclude-test-images**  |    No    | Exclude test images from the output.|
+
+
+### Promote command flags
+
+See the list of flags available for the `promote` command:
```

Comment:
just for my understanding - is this an exhaustive list, or just examples? This sentence implies these are all available commands.

Reply:
These are all available flags that can be used with the `promote` command